src/main/java/step2/view/OutputView.java

```diff
@@ -0,0 +1,72 @@
+package step2.view;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import step2.model.Lotteries;
+
+public class OutputView {
+
+	private static final int DEFAULT_COUNT = 0;
+	private static final int MIN_WIN_COUNT = 3;
+	private static final int MAX_WIN_COUNT = 6;
+	private static final int BENEFIT_STANDARD = 1;
+
+	public void resultView(HashMap<Integer, Integer> finalMatch) {
+		System.out.println("당첨 통계");
+		System.out.println("--------");
+
+		for (int key = MIN_WIN_COUNT; key <= MAX_WIN_COUNT; key++) {
+			showResult(key, isValid(finalMatch.get(key)));
+		}
+	}
+
+	private int isValid(Integer matchSum) {
+		if (matchSum == null) {
+			return DEFAULT_COUNT;
+		}
+		return matchSum;
+	}
+
+	private void showResult(int key, int matchSum) {
+		if (key == MIN_WIN_COUNT) {
+			System.out.println("3개 일치" + "(5000원)- " + matchSum + "개");
+		}
+		if (key == MIN_WIN_COUNT + 1) {
+			System.out.println("4개 일치" + "(50000원)- " + matchSum + "개");
+		}
+		if (key == MIN_WIN_COUNT + 2) {
+			System.out.println("5개 일치" + "(1500000원)- " + matchSum + "개");
+		}
+		if (key == MAX_WIN_COUNT) {
+			System.out.println("6개 일치" + "(2000000000원)- " + matchSum + "개");
+		}
+	}
+
+	public void showTotalTicket(int tickets) {
+		System.out.println(tickets + "매를 구매했습니다");
+	}
+
+	public void showLotteries(int tickets, Lotteries lotteries) {
+		for (int i = 0; i < tickets; i++) {
+			System.out.println(Arrays.toString(lotteries.showLotteries(i).getLotto().toArray()));
+		}
+	}
+
+	public void showRate(double rate) {
+		if (checkRate(rate)) {
```

Comment:
수익률이 손해인지 이득인지를 검증하는 메서드이니 이름을 적절하게 바꿔보는 건 어떨까요?
```suggestion
		if (isProfit(rate)) {
```